website/src/pages/documents/document.page.tsx

```diff
@@ -128,28 +128,31 @@ const TranslationTab = ({ doc }: { doc: Document }) => {
 
   const dialog = useDialogState({ animated: true })
 
-  const [selectedWord, setSelectedWord] =
-    useState<Dailp.FormFieldsFragment | null>(null)
+  const [selectedSegment, setSelectedSegment] = useState<PanelSegment | null>(
+    null
+  )
+
+  const selectAndShowContent = (content: PanelSegment | null) => {
+    setSelectedSegment(content)
 
-  const selectAndShowWord = (content: Dailp.FormFieldsFragment | null) => {
-    setSelectedWord(content)
     if (content) {
       dialog.show()
     } else {
       dialog.hide()
     }
   }
 
-  // When the mobile version of the word panel is closed, remove any word selection.
+  // When the mobile version of the word panel is closed, remove any segment selection.
```

Comment:
Is segment here just referring to a paragraph? I know in other areas of the code, segment refers to a line and paragraphs are called blocks. I could also be misinterpreting what's going on here though

Reply:
Oh, actually, after looking a bit closer, it looks like now either a word or a paragraph could be considered a segment, which makes more sense!

Reply:
Yeah, I think throughout the code, the meaning of segment is pretty flexible. Glad to know it's not too confusing!